manual/easy.install/tools/chocolateyInstall.ps1

```diff
@@ -115,8 +103,7 @@ function setup-python() {
   Install-ChocolateyPath $python_script 'User'
 
   Write-Host "Setting PYTHONHOME environment variable to '$python_home'"
-  Write-Host "PS: PYTHONHOME variable is not required to Python works, but it is a good practice to have it."
-  [Environment]::SetEnvironmentVariable('PYTHONHOME', $python_home, 'User')
+  [Environment]::SetEnvironmentVariable('PYTHONHOME', $python_home, 'User')  

```

Comment:
Probably not a bad idea to use what Chocolatey has for this instead. I'll make that change.